cms/envs/common.py

```diff
@@ -868,7 +868,8 @@
     'django_nose',
 
     # For CMS
-    'contentstore',
+    'contentstore.apps.ContentstoreConfig',
```

Comment:
This is based on a recommendation/mandate from the django docs: https://docs.djangoproject.com/en/1.10/ref/applications/#configuring-applications